src/raylib.h

```diff
@@ -1239,8 +1239,7 @@ RLAPI bool ExportImageAsCode(Image image, const char *fileName);
 
 // Image generation functions
 RLAPI Image GenImageColor(int width, int height, Color color);                                           // Generate image: plain color
-RLAPI Image GenImageGradientV(int width, int height, Color top, Color bottom);                           // Generate image: vertical gradient
-RLAPI Image GenImageGradientH(int width, int height, Color left, Color right);                           // Generate image: horizontal gradient
+RLAPI Image GenImageLinearGradient(int width, int height, int direction, Color start, Color end);        // Generate image: linear gradient
```

Comment:
To be aligned with `GenImageGradientRadial()`, this function should be named `GenImageGradientLinear()`
Also, direction parameter should be detailed in function description.

Reply:
Added a comment and renamed the function